warehouse/src/main/java/io/skai/warehouse/service/impl/ComponentServiceImpl.java

```diff
@@ -40,19 +41,27 @@ public List<ComponentDto> create(List<ComponentDto> components) {
 
     @Override
     public Boolean updateResidues(List<ComponentDto> components) {
-        List<ComponentDto> dtosToWrite = calculateResidues(components);
-        Long wrongDtoCount = getCountOfNegative(dtosToWrite);
+        List<ComponentDto> decrementedResidues = calculateResidues(components);
+        Long countWithNegativeResidues = getCountOfNegative(decrementedResidues);
 
-        if (wrongDtoCount == 0) {
-            List<InsertOnDuplicateUpdateComponentCommand> commands =
-                    componentCommandsBuilder.buildUpsertCommands(dtosToWrite);
+        if (countWithNegativeResidues == 0) {
+            List<InsertOnDuplicateUpdateComponentCommand> commands = componentCommandsBuilder.buildUpsertCommands(decrementedResidues);
 
             componentPersistence.insertOnDuplicateUpdate(commands);
             return Boolean.TRUE;
         }
         return Boolean.FALSE;
     }
 
+    @Override
+    public ResponseEntity<Boolean> updateResiduesAndWrapToResponseStatus(List<ComponentDto> components) {
```

Comment:
updateResidues is the only relevant action so it's sufficient to be used as a name.  AndWrapToResponseStatus - understandable from context, no need to highlight it, never.

Reply:
i fave such method, that updates residues and return Bolean. So i wanted to find correct name for method that calsUpdateMethod and wrapIt into response entity